web/src/css/6-components/_know-your-rights-route.scss

```diff
@@ -0,0 +1,21 @@
+.RightsList {
+    font-family: Times;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    padding: 0.6rem;
+}
+
+.RightsItem h6 {
+    font-weight: bold;
+    font-size: 1.2em;
+    padding: 0.6rem;
+    display: block;
```

Comment:
I don't think you need to do `display: block;` here? Also there's an extra newline.